### ISSUE TITLE
Docs/warnings unclear about accepted way to create pendulum instance from parsed string

### ISSUE THREAD

**Issue by dlopuch** [sdispater/pendulum#198]
Testing out the lib and I'm unclear as to the accepted non-deprecated way of creating a pendulum date from a custom-format parsed string.

Docs say to use `from_format()`, but that's deprecated:

```
>> pendulum.from_format('2018-03-10 23:00:00', '%Y-%m-%d %H:%M:%S', 'US/Pacific')
/bin/ipython:1: PendulumDeprecationWarning: Using the classic formatter in from_format() is deprecated and will no longer be possible in version 2.0. Use the alternative formatter instead.
```

Okay, lets say I use the alternative formatter:

```
>> pendulum.from_format('2018-03-10 23:00:00', 'YYYY-MM-DD HH:mm:ss', 'US/Pacific', formatter='alternative')
lib/python3.6/site-packages/pendulum/pendulum.py:483: PendulumDeprecationWarning: The create() helper will no longer exist in version 2.0. Use datetime() instead.
  return cls.create(**actual_parts)
```

Using pendulum.datetime can't parse a custom string -- designed for positional arguments -- so not clear what that warning is trying to say.

Am I missing something in the docs?  Whats the accepted non-deprecated way to create pendulum instances from a parsed string?

**Comment by sdispater** [comment]
No, you are not missing anything. I am at fault here. `from_format()` uses `create()` internally, which it shouldn't, and triggers the warning.

But your second example is the correct one since this will be the only formatter available in the future `2.0` version.

**Comment by Secrus** [comment]
Closing, since this concerns versions before 2.x.
